transformer_engine/pytorch/softmax.py

```diff
@@ -205,8 +205,8 @@ def __init__(
         self,
         attn_mask_type: str,
         mask_func: Callable,
-        softmax_in_fp32: bool,
         scale: float,
+        softmax_in_fp32: bool = True,
```

Comment:
Why did you change the order?

Reply:
To make `softmax_in_fp32` optional as opposed to completely removing it